erts/doc/src/erlang.xml

```diff
@@ -7473,38 +7465,63 @@ lists:map(
         </list>
         <p>Notice that a scheduler can also be busy even if the
           OS has scheduled out the scheduler thread.</p>
-        <p>Returns <c>undefined</c> if system flag
+
+        <note><p>It is recommended to use the module
+          <seeerl marker="runtime_tools:scheduler"><c>scheduler</c></seeerl>
+          instead of this function directly as it provides an easier way to
+          get the information that you usually want.</p></note>
+
+        <p>If <seeerl marker="#system_flag_scheduler_wall_time">enabled</seeerl>
+          this function returns a list of tuples with
+          <c>{<anno>SchedulerId</anno>, <anno>ActiveTime</anno>,
+          <anno>TotalTime</anno>}</c>, where <c><anno>SchedulerId</anno></c>
+          is an integer ID of the scheduler, <c><anno>ActiveTime</anno></c> is
+          the duration the scheduler has been busy, and
+          <c><anno>TotalTime</anno></c> is the total time duration since
           <seeerl marker="#system_flag_scheduler_wall_time">
-          <c>scheduler_wall_time</c></seeerl> is turned off.</p>
-        <p>The list of scheduler information is unsorted and can
-          appear in different order between calls.</p>
-	<p>As of ERTS version 9.0, also dirty CPU schedulers will
-	be included in the result. That is, all scheduler threads
-	that are expected to handle CPU bound work. If you also
-	want information about dirty I/O schedulers, use
-	<seeerl marker="#statistics_scheduler_wall_time_all">
-        <c>statistics(scheduler_wall_time_all)</c></seeerl>
-	instead.</p>
```

Comment:
The rewrite of this paragraph also removes the reason to why dirty IO schedulers is not included in the result.